addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -85,9 +85,9 @@ export class ColorBlindness extends Component<ColorBlindnessProps, ColorBlindnes
       id: i,
```

Comment:
is this `id` used anywhere?

Reply:
suggest:
```js
id: i.toLowerCase()
...
  this.setFilter(this.id)
```

Reply:
after I modified the array of types, its still lowercase and only the first char is capitalized in the title. Not sure if the id is being used but I can check.